source/sessions/tests/server-support.yml

```diff
@@ -0,0 +1,102 @@
+runOn:
+  - minServerVersion: "3.6.0"
+    topology: ["single"]
+  - minServerVersion: "4.0"
+    topology: ["replicaset"]
```

Comment:
What about these tests requires a 4.0 replica set or 4.1.8 sharded cluster?

We're not using fail points, so this looks like basic session support that would be 3.6+. Let me know if I'm missing something.

Reply:
fixed